shared/common-adapters/usernames.js

```diff
@@ -72,6 +72,23 @@ const nonInlineStyle = {
 }
 const inlineProps = isMobile ? {lineClamp: 1} : {}
 
+// TODO: team and channels
+function matchesFilter(name: string, filter: string): boolean {
+  if (!filter) {
+    return true
+  }
+
+  // No need to worry about Unicode issues with toLowerCase(), since
+  // names can only be ASCII.
+  return name.toLowerCase().indexOf(filter.toLowerCase()) >= 0
+}
+
+const sortNamesWithFilter = (names: Array<string>, filter: ?string): Array<string> => {
```

Comment:
Ahhh, I had thought that 'jack,cjb,miles,mark' in https://app.zeplin.io/project/56b25ffc64fc91773a73aaf8/screen/598a3926676fff2f2ab6f780 was 'jack,cjb,miles,mark,...,malg', and it turned into 'jack,cjb,miles,malg'. Will fix.